views/components/about-tab.pug

```diff
@@ -0,0 +1,15 @@
+div(id='about-tab', class='col s12')
+  div(class='content-container')
+    h5(style="margin: 20px; text-align: center") Below are a handful of scientific articles providing information on the negative cardiovascular effects of sitting.
+    h6(style="margin: 20px; text-align: center; color: grey") Keep in mind, this is not an exhaustive list. 
+    ul(style="margin: 20px")
+      li(style="margin: 10px 0")
```

Comment:
Everything here should be using `class` to format, not `style`(that way you only need to change the style in one place - the css class)